blockchain/storage/leveldb/leveldb.go

```diff
@@ -4,10 +4,51 @@ import (
 	"fmt"
 
 	"github.com/dogechain-lab/dogechain/blockchain/storage"
+	"github.com/dogechain-lab/dogechain/helper/common"
 	"github.com/hashicorp/go-hclog"
 	"github.com/syndtr/goleveldb/leveldb"
+	"github.com/syndtr/goleveldb/leveldb/filter"
+	leveldbopt "github.com/syndtr/goleveldb/leveldb/opt"
 )
 
+const (
+	// minCache is the minimum memory allocate to leveldb
+	// half write, half read
+	minCache = 16
+
+	// minHandles is the minimum number of files handles to leveldb open files
+	minHandles = 16
+
+	DefaultCache   = 16  // 16 MiB
+	DefaultHandles = 512 // files handles to leveldb open files
+
+	DefaultBloomKeyBits = 16 // bloom filter bits
```

Comment:
What should this default value used for?

Our hash value would be 32 bytes. Isn't it better to use a larger bits (e.g. `24`) for less hash collision?

Reply:
11 bit is leveldb default value, 16 is 2 bytes(short int), maybe 2048 bits (256 bytes) is better (ethereum use)

Reply:
Yes, more bits means more hash space and less collisions. But in the same time, more bits means less compression ratio and more disk spaces.

Would you please give out more data on this parameter tuning?

Reply:
Need to analyse the key length and number of keys for a single SST block [bloom-calculator](https://www.di-mgt.com.au/bloom-calculator.html)

Reply:
Could you provide the manual test result in the PR description?